_posts/2020-10-01-intro-developer-advocate.md

```diff
@@ -0,0 +1,111 @@
+runtime and slow query turnaround. Eventually, our architect introduced our team
+to Presto as an alternative, and I worked with him to start testing and moving
+our existing use cases built on Hive to use Presto. We also used Elasticsearch
+and had a few cases that needed to perform joins and unions over the datasets in
+both Elasticsearch and Hive. There were a few use cases that were not going to
+immediately be transferable without some modification to the Elasticsearch
+connector. The first was [adding support for the Elasticsearch array 
+types](https://github.com/prestosql/presto/issues/2441), and the second was, 
+[support for nested types](https://github.com/prestosql/presto/issues/754). 
+
+My first interaction with the Presto community was incredible! As a serial
+open-source attempter, I had always wanted to get invested in an open-source
+project. I had started pull requests in various projects. Sometimes I ran into 
+jerks, in other cases the rules were daunting or too confusing to start, or I
+created a pull request only to have it sit there with no communication
+as to why it wasn't accepted or even looked at. However, when I first joined the
+Slack channel I searched to see if there was already a discussion about array
+types in the history. I ran into [a discussion between Dain and Martin about
+this issue](https://prestosql.slack.com/archives/CP1MUNEUX/p1570064139005900). 
+I conversed with Martin, who was incredibly polite and willing to take time to 
+discuss how this should be implemented. When I actually pulled the code, I saw
+how well written and maintained it was compared to many open-source projects I
+had seen in the past. I made a few changes, wrote a test around my use case, and
+signed a CLA agreement. After a couple of weeks, my pull request was merged
+and I had finally contributed to open-source. After that interaction and seeing
+the code I wanted to do more. I really saw something special with this
+community.
+
+While many Presto contributors are doing amazing work contributing code, I
+noticed there were some holes in other areas of the community that needed to be
+filled. I started answering questions on Slack, LinkedIn, and Twitter and I
+planned out a Udemy course for Presto. The [initial 
+video](https://youtu.be/RPaG0Gu2I6c) I piloted was about how to tune memory in
+ Presto.
+
+Around this time I got into contact with some folks at Starburst about joining 
+them to work with the community and Presto full-time! As I joined we hadn't
+figured out what my exact role was at Starburst but eventually decided I would
+best serve as a Developer Advocate in Developer Relations. If you’re like 
+me, you haven't heard about Developer Relations and only heard of the term 
+Developer Advocate. What I've come to find is this role is aiming to do exactly
+what I set out to do before I joined. As a Developer Advocate, I serve the
+community and act as a liaison between Starburst and the Presto community. Up
```

Comment:
Maybe we should also add about interaction within the community and helping everyone become a part of the community

Reply:
The health of a community is difficult to define and is generally
subject to various signals that we can observe. These signals include an
increase in helpful interactions within the community, new members joining the
community, members who are actively engaging in the community, diversity of the
community, and more. If we start by focusing on making the community successful,
the success of the project will follow. Keeping the goal in mind that co-creator
David Phillips mentions:

> This is the type of project that we look at Postgres as the inspiration. 
> Postgres started in the eighties, it became a SQL system in the nineties, and
> it's still in active use and active development today. We say we want Presto
> to have the same kind of history. - David Phillips